src/container/shared.scss

```diff
@@ -19,6 +19,10 @@ $header-focus-visible-padding: calc(#{awsui.$space-scaled-s} - #{awsui.$border-d
   @include styles.container-shadow;
 }
 
+@mixin borders {
+  @include styles.container-border;
```

Comment:
It's not quite clear why this mixin wouldn't just be part of the @borders-and-shadows mixin above? Or why either of these mixins are needed, as they are just wrappers for individual mixins.